scripts/verify-storage-modes.sh

```
@@ -0,0 +1,417 @@
+    echo -e "${C}║${W}  $1${C}$(printf '%*s' $((62 - ${#1})) '')║${NC}"
+    echo -e "${C}╚══════════════════════════════════════════════════════════════════╝${NC}"
+}
+
+step() { echo -e "\n${M}▶ $1${NC}"; }
+ok() { echo -e "  ${G}✓ $1${NC}"; }
+fail() { echo -e "  ${R}✗ $1${NC}"; }
+info() { echo -e "  ${B}ℹ $1${NC}"; }
+
+wait_for_api() {
+    local timeout=90
+    local start=$(date +%s)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)